app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+
+    people: dict[str, "Person"] = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    if not people:
+        return []
+    list_person = [Person(human["name"], human["age"]) for human in people]
+    for human in people:
+        if human.get("wife"):
+            Person.people[human["name"]].wife = Person.people[human["wife"]]
+        elif human.get("husband"):
+            Person.people[human["name"]].husband \
+                = Person.people[human["husband"]]
```

Comment:
The line continuation character `\` is unnecessary here and could be removed for better readability. The assignment can be done in a single line without breaking it.

Reply:
But the line will be too long